action.yaml

```diff
@@ -10,23 +10,16 @@ inputs:
   aws-secret-key:
     description: 'AWS secret key to create perfomance tests environment'
     required: true
-  adapter:
-    description: 'Artipie adapter to test'
-    required: true
-  scenario:
-    description: 'JMeter scenario file to execute'
-    required: true
```

Comment:
@Sammers21 why to remove these inputs? We may need to run benchmarks only for working adapters and

Reply:
@g4s8, `entry-point.py` has `directories` variable. Benchmarks to run is listed here. Each time we create a new benchmark, we update the list.